charts/incubator/amd-automatic-music-downloader/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+appVersion: "latest"
+dependencies:
+- name: common
+  repository: https://truecharts.org
+  version: 8.15.2
+deprecated: false
+description: A Lidarr companion script to automatically download music for Lidarr.
+home: https://github.com/truecharts/apps/tree/master/charts/incubator/amd-automatic-music-downloader
+icon: https://truecharts.org/_static/img/appicons/amd-automatic-music-downloader-icon.png
+keywords:
+- amd
+- automatic
+- music
+- downloader
+- deemix
+kubeVersion: '>=1.16.0-0'
+maintainers:
+- email: info@truecharts.org
+  name: TrueCharts
+  url: https://truecharts.org
+name: amd-automatic-music-downloader
```

Comment:
Maybe we changed it to `automatic-music-downloader` ?
As this is the name will be displayed to the GUI. `amd` part might be mistaken with the `amd-plugin`?

Reply:
Also rename folder, app icon and `home` link in Charts.yaml